### PR TITLE
Fix issue with React Native 56.0

### PR DESCRIPTION
**Pull request by Suraj-Tiwari**

### CODE REVIEW

File: android/src/main/java/com/sbugert/rnadmob/RNAdMobRewardedVideoAdModule.java [hunk 1]
```diff
@@ -90,6 +90,11 @@ public void onRewardedVideoAdLeftApplication() {
     public void onRewardedVideoCompleted() {
         sendEvent(EVENT_VIDEO_COMPLETED, null);
     }
+    
+    @Override
+    public void onRewardedVideoCompleted() {
```
Comment (koenpunt) [inline android/src/main/java/com/sbugert/rnadmob/RNAdMobRewardedVideoAdModule.java]: This method is already implemented, so should not be added again.

File: android/build.gradle [hunk 2]
```diff
@@ -1,12 +1,12 @@
 apply plugin: 'com.android.library'
 
 android {
-    compileSdkVersion 23
-    buildToolsVersion "23.0.1"
+    compileSdkVersion 26
+    buildToolsVersion "26.0.1"
```
Comment (koenpunt) [inline android/build.gradle]: Do you think there's a way to do this conditionally? So projects not using RN 56 do not have to update their SDK versions?

File: android/src/main/java/com/sbugert/rnadmob/RNAdMobRewardedVideoAdModule.java (hunk 1 above)
Comment (Suraj-Tiwari) [reply android/src/main/java/com/sbugert/rnadmob/RNAdMobRewardedVideoAdModule.java]: I think Npm version don't have this. I've faced this issue before but that was my quick workaround for this I'll FIx this.

File: android/build.gradle (hunk 2 above)
Comment (Suraj-Tiwari) [reply android/build.gradle]: Check new Commit. I've Fixed this.

**Comment by Suraj-Tiwari** [comment]
Any update on this?

**Comment by Suraj-Tiwari** [comment]
@sbugert @koenpunt Commits are updated.... Review  it

**Comment by koenpunt** [comment]
This has been fixed on master (and released as `v2.0.0-beta.6`)

**Pull request closed without merging**